src/Lecture1.hs

```diff
@@ -63,7 +65,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integer -> Integer
+lastDigit = flip mod 10 . abs
```

Comment:
Great! I see that you're familiar with function composition 🙂 

To be more explicit about what is happening in this function, you can also use infix notation of functions.

```suggestion
lastDigit n = abs n `mod` 10
```